src/install.rs

```diff
@@ -607,6 +607,9 @@ impl Installer {
 
         self.add_pkg(config, base, repo, &pkgdests, &debug_paths)?;
         self.queue_install(base, &pkgdests, &debug_paths);
+
+        post_build_command(config, dir, base.package_base(), &version)?;
```

Comment:
I think the position is good, only difference to placing this 2 lines up is it will run after adding the package to the local repo. Which some person may find use in and there's no downside.